api/src/test/resources/ORM_O01.hl7

```diff
@@ -1,5 +1,5 @@
 MSH|^!\&||LNSP|||20180330150703.345+{time_offset}00||ORM^O01^ORM_O01|2018033015070300000|D|2.5
-PID||1004X^^^^^2|||Doe^Joe^Terry|Janine||M|||Trou coucou^1ère L'Arbre^Anse Rouge^Artibonite^^HTI^^Anba Morne Porte|||||^Separe|^Vaudou|^5234A^^^^LNSP|||||Birth City
-PV1|||2||||||||||||||PROVIDER_ID^Fami&von^Prov^Ider^^a^dr|||||||||||||||||||||||||||20180425082533.4+{time_offset}00
+PID||1004X^^^^^2|||Doe^Joe^Terry|Janine|19870211[1015]|M|||Trou coucou^1ère L'Arbre^Anse Rouge^Artibonite^^HTI^^Anba Morne Porte|||||^Separe|^Vaudou|^5234A^^^^LNSP|||||Birth City
```

Comment:
Lack of 13th section 'Phone Number - Home' (marked as green in Inbound (HL7 2.5.1 OML^O21)).